### ISSUE TITLE
Warn if the package module is missing a docstring

### ISSUE THREAD

**Issue by ianshmean** [JuliaRegistries/RegistryCI.jl#319]
If a module is missing a docstring and a user tries `help?> Package`, currently nothing shows. https://github.com/JuliaLang/julia/pull/39093 aims to print the README contents

But, it would be good to promote the addition of carefully crafted docstrings for the main module in the package, as they would likely be better crafted for purpose.

Could the merge bot warn if there isn't one?
Perhaps there are other best-practices to check for too?

**Comment by DilumAluthge** [comment]
Yeah, maybe a warning that doesn't block AutoMerge?

**Comment by DilumAluthge** [comment]
Yeah we should also e.g. warn if you don't have a `test/runtests.jl` file.

**Comment by ianshmean** [comment]
Yeah, I don't think it should block. But I do think the warnings should be on both new registrations and new versions

**Comment by KristofferC** [comment]
How many packages have a module docstring? To me, this feels a bit too nitpicky. There could be a `BestPractices.jl` package to run on your code but the registration process seems like the wrong place for a bunch of best practice suggestions. Some best practices are opinionated and most people don't read the comment from the registration bot.
